src/apyfixedarray_wrapper.cc

```diff
@@ -299,6 +299,29 @@ void bind_fixed_array(nb::module_& m)
         array([[ 1. ,  1.5],
                [-2. , -1.5]])
 
+        --------
+             )pbdoc")
+
+        .def("flatten", &APyFixedArray::flatten, R"pbdoc(
+        Reshape the APyFixedArray to the specified shape without changing its data.
+
+        Returns
+        -------
+        :class:`APyFixedArray`
+        Raises
```

Comment:
There needs to be an empty line here (and in reshape above) as this renders a bit weird otherwise, see: https://apytypes.github.io/apytypes/api/apyfixedarray.html#apytypes.APyFixedArray.reshape